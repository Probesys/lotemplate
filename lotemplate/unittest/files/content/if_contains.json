{
  "tutu": {
    "type": "text",
    "value": "text content with searchable words"
  }
}