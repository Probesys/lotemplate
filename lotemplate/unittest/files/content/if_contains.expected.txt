
Tests of if CONTAINS statements

We check the values with the tutu = text content with searchable words


This paragraph should be displayed (contains the « content » word)



This paragraph should be displayed (contains case insensitve)



This paragraph should be displayed (substring inside a word)


<-nothing


This paragraph should be displayed (does not contain glouglou)


<-nothing

end of file
